com.pilosa.client/src/main/java/com/pilosa/client/Internal.java

```diff
@@ -5744,13 +5761,13 @@ public int getSerializedSize() {
         size += com.google.protobuf.CodedOutputStream
                 .computeBoolSize(5, remote_);
       }
-      if (excludeAttrs_ != false) {
+        if (excludeRowAttrs_ != false) {
         size += com.google.protobuf.CodedOutputStream
-          .computeBoolSize(6, excludeAttrs_);
-      }
-      if (excludeBits_ != false) {
+                .computeBoolSize(6, excludeRowAttrs_);
+        }
+        if (excludeColumns_ != false) {
```

Comment:
though in this particular place the plural sounds more appropriate.

Reply:
Likewise, automatically generated.